__tests__/app.test.js

```diff
@@ -107,4 +107,40 @@ describe("/api", () => {
           expect(body.articles).toBeSortedBy("created_at", { coerce: true });
         });
     });
+  });
+
+  describe("/api/articles/:articleId/comments", () => {
+    test("GET:200 sends an array of all comments on the specified article", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.comments.length).toBe(11);
+          body.comments.forEach((comment) => {
+            expect(typeof comment.comment_id).toBe("number");
+            expect(typeof comment.votes).toBe("number");
+            expect(typeof comment.author).toBe("string");
+            expect(typeof comment.body).toBe("string");
+            expect(typeof comment.created_at).toBe("string");
+            expect(typeof comment.article_id).toBe("number");
```

Comment:
We can be a lot more specific here by specifying what article_id we expect the comments to have!